src/components/app-list/index.js

```diff
@@ -78,44 +78,50 @@ export class AppList extends React.Component {
             <AppListItem app={app} key={app.name} handler={this.handler} />
           ))
       ) : (
-        <Typography variant="body_short">No favourites</Typography>
+        <Typography>No favourites</Typography>
       );
 
     return (
-      <article className="app-list">
+      <article className="grid grid--gap-medium">
+        <div className="app-list__header">
+          {apps.length ? (
```

Comment:
suggestion: change for readability
```suggestion
          {apps.length > 0 ? (
```

Reply:
fixed